content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,69 @@
+# Machine Learning Model Registry
```

Comment:
Should we call this **Git-native Model Registry** or **MRaC** (MR as code) instead? In the future

Reply:
A bit too complicate. Business use case is "Model Registry" - everything else hides it to my mind